Sources/CSQLite/include/sqlite_nio_sqlite3.h

```diff
@@ -2302,7 +2302,7 @@ struct sqlite3_mem_methods {
 ** database handle, SQLite checks if this will mean that there are now no
 ** connections at all to the database. If so, it performs a checkpoint
 ** operation before closing the connection. This option may be used to
-** override this behaviour. The first parameter passed to this operation
+** override this behavior. The first parameter passed to this operation
```

Comment:
Strongly disagree with this change 🤣

Reply:
Hey, I just run the vendoring command, I don't judge the diff 😂